packages/snaps-controllers/src/services/webview/WebViewExecutionService.test.ts

```diff
@@ -0,0 +1,30 @@
+import { createService } from '../../test-utils';
+import { WebViewExecutionService } from './WebViewExecutionService';
+
+class WebViewExecutionServiceWrapper extends WebViewExecutionService {
+  public async testInitEnvStream(jobId: string) {
+    return this.initEnvStream(jobId);
+  }
+}
+
+const mockedWebView = {
+  addEventListener: jest.fn(),
+  removeEventListener: jest.fn(),
+  injectJavaScript: jest.fn(),
+};
+
+describe('WebViewExecutionService', () => {
+  jest.useRealTimers();
+
+  beforeEach(() => {
+    mockedWebView.addEventListener.mockReset();
```

Comment:
Do we not already do this by default?

Also there is only 1 test so it seems unnecessary

Reply:
Removed.